FEMComputationalFluidDynamicsAnalysis/postProcessing/computeForces.m

```diff
@@ -0,0 +1,45 @@
+function Force = computeForces (FComplete, analysis, parameters, nodesDomain)
+%% Licensing
+%
+% License:         BSD License
+%                  cane Multiphysics default license: cane/license.txt
+%
+% Main authors:    Marko Leskovar
+%                  Andreas Apostolatos
+%
+%% Function documentation
+%
+% Calculate force on a single body
+%
+%               Input :
+%           FComplete : The complete force vector
+%            analysis : Analysis type and number of dimensions
```

Comment:
% analysis : General analysis parameters
% *identation*                                           .type : Analysis type
% *identation*                .noSpatialDimensions : Number of spatial dimensions
% *identation*                                    .noFields : Number of DOFs per node